va/va_enc_av1.h

```diff
@@ -666,7 +666,8 @@ typedef struct  _VAEncPictureParameterBufferAV1 {
      *  0: 16x16 block size, default value;
      *  1: 32x32 block size;
      *  2: 64x64 block size;
-     *  3: 8x8 block size.
+     *  3: 8x8 block size;
+     *  4: 4x4 block size.
```

Comment:
any value is ok? or backend driver only accept part of them? if so , may need a capability attributes

Reply:
Av1 does have a caps report for it. https://github.com/intel/libva/blob/02eed8a57faadcd7f0e5e999723286ed160bd2bd/va/va_enc_av1.h#L154C18-L154C47
This change in comment is for interface completeness. In av1 spec, 4x4 block is allowed. 
From av1 spec: Segmentation map
A 3-bit number containing the segment affiliation for each 4x4 block in the image.